baseband/vlbi_base/base.py

```diff
@@ -313,8 +328,8 @@ def get_time1(self):
     @property
     def size(self):
         """Number of samples in the file."""
-        return int(round((self.stop_time - self.start_time).to(u.s).value *
-                         self.frames_per_second * self.samples_per_frame))
+        return int(np.round(((self.stop_time - self.start_time) *
```

Comment:
Strange that one needs round... That's a buglet in Quantity. But don't use decompose, but rather the unit you want: Maybe `int(((self.stop_time - self.start_time) * self.sample_rate).to(u.one).round())`. (or `u.dimensionless_unscaled` if you want to be more elaborate...)